### ISSUE TITLE
page loader: mention the widget that is failing feature-validation

### ISSUE THREAD

**Issue by stefanguha** [LaxarJS/laxar#293]
When a widget is configured in some page with features that don't validate against the widget's widget.json descriptor, an error message is thrown.

it reads like this: 

```
Validation for widget features failed. Errors: 
 - Additional properties not allowed: openPopupAction. Path: "$.openPopupAction"."
```

The error message does not mention the widget's name (or if available, configured 'id'). That potentially makes it hard to fix the problem.

**Comment by jpommerening** [comment]
Which LaxarJS version are you using?

Usually the exception message should look like this:
```
Error loading widget "initialization-activity" (id: "initializationActivity-id3"): Validation for widget features failed. Errors: 
 - Additional properties not allowed: openPopupAction. Path: "$.openPopupAction".
```

**Comment by x1B** [comment]
This might be a problem with compositions, specifically. We should probably change the error message to say something more helpful like:

```
Error loading composition "my-superb-composition". Validation for composition features failed: 
  - (etc blablabla)
```

**Comment by stefanguha** [comment]
Was using Laxar 1.2.1

**Comment by x1B** [comment]
Fixed on `master` (`v2.0.0`) and `master-1.x` (`v1.3.0`).
